.travis.yml

```diff
@@ -11,12 +11,12 @@ script:
 
 deploy:
 - provider: script
-  script: docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make docker-build docker-push
+  script: GINKGO_EXTRA_ARGS="--compilers=2" make all test/unit && docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io && make docker-build docker-push
```

Comment:
no need to run the tests in the deployment state they run in the `make check`

Reply:
done. I have updated the PR.